webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -5376,6 +5376,9 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
                 if(append && MCK_BOT_MESSAGE_DELAY !== 0 && mckMessageLayout.isMessageSentByBot(msg, contact)) {
                     botMessageDelayClass = 'n-vis';
                 }
+                if (!richText && messageClass == "n-vis"){
+                    return ;
+                }
```

Comment:
Please don't compare it with the hardcoded value.
Use the checks which are there to identify when we will be assigning "n-vis" value to messageClass variable.
+ By reading this check not able to understand why we're returning from this function, adding a comment explaining why we're returning from this function would be helpful for others.